internal/builders/gradle/README.md

```diff
@@ -51,8 +51,9 @@ to send us feedback!
 
 The Gradle builder currently has the following limitations:
 
-1. The project must be build'able by way of `gradle build`. If you need the option for flags, profiles or something else to define more granular builds, please open an issue.
-
+1. The project must be build'able by way of `./gradlew build`. If you need the option for flags, profiles or something else to define more granular builds, please open an issue.
+2. The project must include a gradle wrapper (`gradlew`). The slsa builder does not include an installation of gradle.
```

Comment:
nit: I think the terminology used in the doc is "Gradle builder" to refer to the builder/reusable workflow.
```suggestion
2. The project must include a gradle wrapper (`gradlew`). The Gradle builder does not include an installation of gradle.
```